frontend/src/views/dev_frame/load-mock-data.tsx

```diff
@@ -263,6 +275,126 @@ export function SeedDataMenu({
         }
     }
 
+    async function seedApplications(limit = 600) {
+        setStage("Applications");
+        setProgress(0);
+
+        if (!targetSession) {
+            throw new Error("Need a valid session to continue");
+        }
+
+        let count = 0;
+        let total = seedData.applications.length;
+
+        // Keep track of the original active user so we can swap back
+        const initialUser = await dispatch(fetchActiveUser());
+
+        // Get this session's posting token:
+        const resp = await apiGET(
+            `/admin/sessions/${targetSession.id}/postings`
+        );
+        let url_token = resp[0].url_token;
+
+        for (const application of seedData.applications.slice(0, limit)) {
+            const currUser = {
+                utorid: application.utorid,
+                roles: ["admin", "instructor", "ta"],
+            };
+            await dispatch(
+                debugOnlySetActiveUser(currUser, { skipInit: true })
+            );
+
+            const newApp = {
+                utorid: application.utorid,
+                student_number: application.student_number,
+                first_name: application.first_name,
+                last_name: application.last_name,
+                email: application.email,
+                phone: application.phone.toString(),
+                program: application.program,
+                department: application.department,
+                yip: application.yip,
+                gpa: application.gpa || 0,
+                previous_department_ta: application.previous_department_ta,
+                previous_university_ta: application.previous_university_ta,
+                program_start: application.program_start,
+                previous_other_university_ta:
+                    application.previous_other_university_ta,
+                position_preferences: application.position_preferences,
+            };
+
+            await apiPOST(
+                `/public/postings/${url_token}/submit`,
+                { answers: newApp },
+                true
+            );
+
+            count++;
+            setProgress(Math.round((count / total) * 100));
+        }
+
+        await dispatch(
+            debugOnlySetActiveUser({
+                utorid: initialUser.utorid,
+                roles: initialUser.roles,
+            })
+        );
+
+        setProgress(100);
+    }
+
+    async function seedInstructorPreferences(limit = 600) {
+        setStage("Instructor Preferences");
+        setProgress(0);
+
+        if (!targetSession) {
+            throw new Error("Need a valid session to continue");
+        }
+
+        let count = 0;
+        let total = seedData.applications.length;
+
+        for (const application of seedData.applications.slice(0, limit)) {
+            if (application.instructor_preferences) {
```

Comment:
I prefer early exit rather than putting everything in an if. e.g.

```
if (!application.instructor_preferences) {
    continue;
}
```

then everything else doesn't need to be indented :-)